### PR TITLE
Ria 6593 due date for ada

### PR DESCRIPTION
**Pull request by JamieHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-6593

### Change description ###
For ADA cases once Request Case building event is completed, a standard direction will be sent to LR to build the case within 9 working days.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestCaseBuildingPreparer.java [hunk 1]
```diff
@@ -17,22 +15,30 @@
...
     private final int legalRepresentativeBuildCaseDueInDays;
     private final int legalRepresentativeBuildCaseDueFromSubmissionDate;
+    private final int legalRepresentativeBuildCaseDueInDaysAda;
     private final DateProvider dateProvider;
+    private final DueDateService dueDateService;
 
     public RequestCaseBuildingPreparer(
             @Value("${legalRepresentativeBuildCase.dueInDays}") int legalRepresentativeBuildCaseDueInDays,
             @Value("${legalRepresentativeBuildCase.dueInDaysFromSubmissionDate}") int legalRepresentativeBuildCaseDueFromSubmissionDate,
+            @Value("${legalRepresentativeBuildCase.dueInDaysAda}") int legalRepresentativeBuildCaseDueInDaysAda,
```
Comment (neha-aggarw) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestCaseBuildingPreparer.java:34]: Following the existing naming convention: legalRepresentativeBuildCaseAda.dueInDays would be better here. Second object is to determine the event in specific. But only if you are putting another commit, add this change.

**Review by neha-aggarw — CHANGES_REQUESTED** [review]
Also, please add functional test for this change.

File: src/functionalTest/resources/scenarios/RIA-6593-request-case-building.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "description": "RIA-6593 Request case building direction to legal representative",
```
Comment (neha-aggarw) [inline src/functionalTest/resources/scenarios/RIA-6593-request-case-building.json:2]: This functional test is testing whether the notification is sent out on the completion of event : requestCaseBuilding and not if the due date is appropriate for ADA case. Please have a look at : RIA-892-RIA-2694-request-respondent-evidence-start.json.

**Review by neha-aggarw — DISMISSED** [review]

**Review by neha-aggarw — APPROVED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Pull request merged**